akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -78,7 +83,7 @@ class Http2ClientSpec extends AkkaSpecWithMaterializer("""
           ),
           expectedResponse =
             HPackSpecExamples.FirstResponse
-              .withEntity(Strict(ContentTypes.NoContentType, ByteString.empty))
+              .withEntity(Chunked(ContentTypes.`application/octet-stream`, Source.empty))
```

Comment:
What happened here? That seems wrong for empty entities. We want that to come out as `HttpEntity.Empty`. What is the result for entities with a `content-length: 0` and a different content-type set?

Reply:
Never mind, there are few things weird about how empty entities or responses that do not have entities are parsed but we can fix those later.